react/features/overlay/actionTypes.js

```diff
@@ -0,0 +1,25 @@
+import { Symbol } from '../base/react';
+
+/**
+ * The type of the Redux action which signals that a suspend was detected.
+ *
+ * {
+ *     type: SUSPEND_DETECTED
+ * }
+ * @public
```

Comment:
I don't think we use visibility modifiers for action types.  So far they are always public.